<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_set_pingmubuju</class>
 <widget class="QDialog" name="Dlg_set_pingmubuju">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>屏幕布局</string>
  </property>
  <widget class="QCheckBox" name="m_ckb_wav_eeg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>脑电波</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_xinlv">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>135</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>196</green>
         <blue>136</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>246</red>
         <green>165</green>
         <blue>84</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>67</green>
         <blue>16</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>158</red>
         <green>90</green>
         <blue>22</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>135</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>246</red>
         <green>195</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>135</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>196</green>
         <blue>136</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>246</red>
         <green>165</green>
         <blue>84</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>67</green>
         <blue>16</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>158</red>
         <green>90</green>
         <blue>22</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>135</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>246</red>
         <green>195</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>67</green>
         <blue>16</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>135</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>196</green>
         <blue>136</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>246</red>
         <green>165</green>
         <blue>84</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>67</green>
         <blue>16</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>158</red>
         <green>90</green>
         <blue>22</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>67</green>
         <blue>16</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>67</green>
         <blue>16</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>135</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>135</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>135</green>
         <blue>33</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>心率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_layout">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ecg1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ecg3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_ibp1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_co2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CO2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ecg6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_huxi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>呼吸波</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ecg2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG2</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>115</y>
     <width>120</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ecg5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>15</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>屏幕布局</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>3</width>
     <height>400</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_qushishijian">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>45</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ecg7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_co2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CO2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_xueyang">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>血氧</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_tiwen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>体温</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_mshen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>麻醉深度</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ibp2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_nibp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>无创血压</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_huxilv">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>呼吸率</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_maibo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>脉搏波</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>趋势时间</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ibp1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_ecg4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_ibp2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckb_wav_mashen">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>麻醉深度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>麻醉深度分辨率</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_mashen_fenbianlv">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
